src/QsCompiler/Tests.Compiler/CompletionParsingTests.fs

```diff
@@ -500,7 +507,34 @@ let ``Operation statement parser tests`` () =
         [
             ("repeat ", [])
             ("within ", [])
-            ("using ", [])
+            ("use ", [ Declaration ])
```

Comment:
Are using and borrowing removed from the list of suggested keywords we give for code completion?

Reply:
No, it suggests both `use` and `using`, and `borrow` and `borrowing`.

Reply:
I removed `using` and `borrowing` from code completion.

Reply:
Thanks!